core/templates/pages/skill-editor-page/editor-tab/skill-concept-card-editor/skill-concept-card-editor.directive.ts

```diff
@@ -54,9 +56,10 @@ angular.module('oppia').directive('skillConceptCardEditor', [
         '/pages/skill-editor-page/editor-tab/skill-concept-card-editor/' +
         'skill-concept-card-editor.directive.html'),
       controller: [
-        '$scope', '$filter', '$uibModal', 'EVENT_SKILL_REINITIALIZED',
-        function($scope, $filter, $uibModal, EVENT_SKILL_REINITIALIZED) {
+        '$scope', '$filter', '$uibModal',
+        function($scope, $filter, $uibModal) {
           var ctrl = this;
+          ctrl.parentSubscription = new Subscription();
```

Comment:
Maybe `skillChangeSubscription` or I was thinking along the lines of `subscriptionsTracker`?

Reply:
Probably the former, in terms of format. The latter isn't so good because it sounds like a "tracker" object whereas this is clearly a "subscription" object.

But let's dig into the semantic meaning. Why "skill change"? Do initialization/reinitialization events fire only on a change of the skill ID?

Reply:
Changed this to `directiveSubscriptions`